theano/sandbox/cuda/tests/test_opt.py

```diff
@@ -291,6 +291,27 @@ def test_local_gpu_subtensor():
     assert any([isinstance(node.op, cuda.GpuElemwise) for node in topo])
 
 
+def test_local_split():
+    """ Test that the GpuSplit op is being applied and works """
+    # Construct symbolic split
+    x = tensor.vector()
```

Comment:
It is this one that should be fvector. Currently it is floatX, but for tests, it can be anything float32 or float64